frontend/src/data/variables/AcsPopulationProvider.ts

```diff
@@ -96,7 +100,9 @@ class AcsPopulationProvider extends VariableProvider {
 
     df = this.removeUnwantedDemographicTotals(df, breakdowns);
 
-    // TODO - rename state_fips and county_fips to fips
+    df = df.dropSeries(["ingestion_ts"]).resetIndex();
```

Comment:
Add a TODO to drop this once we make the server no longer send it down (I'll file an issue for removing it on the server)